packages/sdk/src/protocols/filter.ts

```diff
@@ -326,7 +427,7 @@ class FilterSDK extends BaseProtocolSDK implements IFilterSDK {
   ) {
     super(
       new FilterCore(
-        async (pubsubTopic: PubsubTopic, wakuMessage: WakuMessage) => {
+        async (pubsubTopic, wakuMessage, peerIdStr) => {
           const subscription = this.getActiveSubscription(pubsubTopic);
```

Comment:
Wow, I just noticed - we keep only one subscription per `pubsubTopic`. 
That mean that if another one will be created in the application - first one would be dropped. 

Also we probably still have this one https://github.com/waku-org/js-waku/issues/1678 

Let's follow up if I am not wrong

Reply:
oh right, there is a fork here - https://github.com/waku-org/js-waku/blob/128dd587d63793661e92c321360d05bdd651d3c2/packages/sdk/src/protocols/filter.ts#L502

if a subscription exists - then we return it which will be used for `subscribe` request